src/facade/reply_builder.h

```diff
@@ -23,16 +23,43 @@ enum class ReplyMode {
 
 class SinkReplyBuilder {
  public:
-  struct ResponseValue {
-    std::string key;
-    std::string value;
+  struct MGetStorage {
+    MGetStorage* next = nullptr;
+    char data[0];
+  };
```

Comment:
nit: I was once wondering why C++ doesn't have flexible array members and what to use instead, and I remember that zero sized arrays are forbidden by the standard (my gcc argues only with -wpedantic, so it's considered safe). The alternative is using char[1]

Reply:
ok, but you will take this char from your next salary.

Reply:
<img src="https://cdn3.emoji.gg/emojis/9109_Sad_Cat_Thumbs_Up.png" height="200px">